core/tee/tee_rpmb_fs.c

```diff
@@ -322,6 +322,11 @@ static void bytes_to_u16(uint8_t *bytes, uint16_t *u16)
 	*u16 = (uint16_t) ((*bytes << 8) + *(bytes + 1));
 }
 
+static void get_op_result_bits(uint8_t *bytes, uint16_t *u16)
```

Comment:
No need for 16 bits here since op_result is 7 bits => `s/uint16_t *u16/uint8_t *res/

Reply:
Done